src/csharp/Grpc.Tools/build/_grpc/_Grpc.Tools.targets

```diff
@@ -22,6 +22,10 @@
   <Target Name="gRPC_ResolvePluginFullPath" AfterTargets="Protobuf_ResolvePlatform">
     <PropertyGroup>
       <!-- TODO(kkm): Do not use Protobuf_PackagedToolsPath, roll gRPC's own. -->
+
+      <!-- First try environment variable. -->
+      <gRPC_PluginFullPath Condition=" '$(gRPC_PluginFullPath)' == '' ">$(GRPC_TOOL_PLUGIN)</gRPC_PluginFullPath>
```

Comment:
The newly added logic looks good, but I think we should use a different name for the env variable.
How does these options sound?
- `GRPC_TOOLS_PLUGIN` (at least the prefix matches the nuget name)
- `GRPC_TOOLS_PROTOC_PLUGIN` (even more specific)
- `GRPC_PROTOC_PLUGIN` (describes well what the thing is)
- `PROTOBUF_PROTOC_PLUGIN` (since `PROTOBUF_PROTOC` is the env variable used for protoc binary - the name isn't great, but at least this would be consistent with the existing setting)

I'm leaning towards `GRPC_PROTOC_PLUGIN`.